PlayerControls/sources/DefaultControlsViewController.swift

```diff
@@ -78,6 +78,15 @@ public final class DefaultControlsViewController: ContentControlsViewController
         }
     }
     
+    public override func viewWillAppear(_ animated: Bool) {
+        setupAirPlayCurrentVisibility()
+        NotificationCenter.default.addObserver(self, selector: #selector(setupAirPlayCurrentVisibility), name: .MPVolumeViewWirelessRoutesAvailableDidChange, object: nil)
+    }
+    
+    public override func viewWillDisappear(_ animated: Bool) {
```

Comment:
If you override this method, you must call super at some point in your implementation.

Reply:
👍